src/sentry/models/organizationmapping.py

```diff
@@ -20,7 +21,7 @@ class OrganizationMapping(Model):
     * Safely reserve organization slugs via an eventually consistent cross silo workflow
     """
 
-    __include_in_export__ = True
+    __relocation_scope__ = RelocationScope.Organization
```

Comment:
This should be Global or Excluded too. Mapping records would be generated as organizations are imported.

Reply:
Is it done via a signal or something similar?

The reason I put it in `Organization` is because I would like for importers to be able to specify a slug map ("when importing my JSON, convert all instance of org slug `is-taken` to `is-not-yet-taken`"). Since `OrganizationMapping` has slugs, I wanted this to be included in the `Organization`-scoped imports. But if it will be updated automatically from the data on the `slug` field in the `Organization` model, then we can just exclude?

Though on second thought, maybe we keep it anyway, since only the `slug` field on `OrganizationMapping` is `unique=True`? Otherwise I could import an `Organization` with `slug=is-already-taken`, and the import will fail due to a `UniqueViolation`.